android/build.gradle

```diff
@@ -123,80 +118,84 @@ android {
         plus1.res.srcDirs = ['res-plus1']
     }
 
-    lintOptions {
-        abortOnError false
-    }
-
-    dexOptions {
-        javaMaxHeapSize "2g"
-        jumboMode = true
-    }
-
-    signingConfigs {
-        release {
-            storeFile file("../../frostwire-tools/certs/android.keystore/frostwire.keystore")
-            storePassword ""
-            keyAlias ""
-            keyPassword ""
-        }
-    }
-
     buildTypes {
         release {
             minifyEnabled true
             useProguard false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'),
-                    'proguard-rules.pro'
-            signingConfig signingConfigs.release
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+//            signingConfig signingConfigs.release
             multiDexKeepFile file('multidex-config.txt')
             //multiDexKeepProguard file('proguard-rules.pro')
         }
         debug {
             minifyEnabled true
             useProguard false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'),
-                    'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
             multiDexKeepFile file('multidex-config.txt')
             //multiDexKeepProguard file('proguard-rules.pro')
         }
     }
 
-
     flavorDimensions "classicApp"
-
     productFlavors {
         basic {
-            applicationId 'com.frostwire.android'
             versionCode 9060000 + project.ext.versionCode
         }
         plus1 {
-            applicationId 'com.frostwire.android'
             versionCode 9070000 + project.ext.versionCode
         }
     }
 
     applicationVariants.all { variant ->
         changeApkOutput(variant)
     }
+
+    lintOptions {
+        abortOnError false
+    }
+
+    compileOptions {
+        encoding = 'UTF-8'
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    dexOptions {
+        javaMaxHeapSize "2g"
+        jumboMode = true
+    }
+
+    signingConfigs {
+        release {
+            storeFile file("../../frostwire-tools/certs/android.keystore/frostwire.keystore")
+            storePassword ""
+            keyAlias ""
+            keyPassword ""
+        }
+    }
 }
 
 dependencies {
+    implementation 'com.android.support:animated-vector-drawable:28.0.0'
```

Comment:
I'll test your branch, but I had way too many issues when I started moving things to 28 and it was time to build a production APK